src/index.css

```diff
@@ -117,7 +117,7 @@ select {
 }
 
 button {
-  max-width: 29%;
+  max-width: 100%;
```

Comment:
This may affect the buttons for the mailing list download

Reply:
They seem to be maintaining their regular width.